lib/api.c

```diff
@@ -369,3 +369,14 @@ EXP int CALL set_prochot_deassertion_ramp(ryzen_access ry, uint32_t value) {
 		break;
 	}
 }
+
+EXP int CALL set_skin_temp(ryzen_access ry, uint32_t value){
+	value *= 256;
```

Comment:
@FlyGoat I don't know why Renoir does need this multiplier and not sure if Picasso does have a skin temp setting at all.

But for Renoir it does work great.